docs/config/01-overview.md

```diff
@@ -273,3 +273,37 @@ taskQueue: 'async'
 ## testing
 
 Please see the [testing config docs](/testing-config).
+
+## transformAliasedImportPaths
+
+*default: `false`*
+
+This sets whether or not Stencil should transform [path aliases](
+https://www.typescriptlang.org/docs/handbook/module-resolution.html#path-mapping) set
+in a project's `tsconfig.json` from the assigned module aliases to resolved
+relative paths. This will not transform external imports (like `@stencil/core`) or
+relative imports (like `'../utils'`).
+
+This option applies globally and will affect all code emitted by Stencil,
+including `.d.ts` files.
+
+An example of path transformation could look something like:
+
+```ts
+// Source code (ts)
```

Comment:
FWIW - this could be combined into one line with the `title` attribute:
```suggestion
```ts title="Source Code (.ts)"
```
But, that would require us to break this out into three separate blocks:
1. Source Code
2. Output Code
3. Type Declaration

Reply:
ooo I forgot about this functionality w/ docusaurus!

Reply:
I turned it into three codeblocks, added a little more inline explanation, and added a `tsconfig.json` example as well